easybuild/easyconfigs/a/ABAQUS/ABAQUS-2023.eb

```diff
@@ -0,0 +1,31 @@
+name = 'ABAQUS'
+version = '2023'
+
+homepage = 'https://www.3ds.com/products/simulia/abaqus'
+description = """Finite Element Analysis software for modeling, visualization and best-in-class implicit and explicit
+ dynamics FEA."""
+
+toolchain = SYSTEM
+
+sources = [
+    '%(version)s.AM_SIM_Abaqus_Extend.AllOS.1-5.tar',
+    '%(version)s.AM_SIM_Abaqus_Extend.AllOS.2-5.tar',
+    '%(version)s.AM_SIM_Abaqus_Extend.AllOS.3-5.tar',
+    '%(version)s.AM_SIM_Abaqus_Extend.AllOS.4-5.tar',
+    '%(version)s.AM_SIM_Abaqus_Extend.AllOS.5-5.tar',
+]
+
+
+download_instructions = f"""{name} requires manual download from https://www.3ds.com/support/software-downloads.
+
+Required downloads:\n{chr(10).join(sources)}"""
```

Comment:
This targets the `develop` branch and uses f-strings, so it'll have to wait until we collapse he 5.0.x branch into develop.

Reply:
Yeah I set the 5.0 milestone for that reason
No rush getting it in, just gonna make the test builds anyway because this still works unless any one of the maintainers are actually using python 2